src/screens/WebView/PrivacyPolicies.js

```diff
@@ -1,36 +1,37 @@
 import * as React from 'react';
-import { ActivityIndicator, SafeAreaView, StyleSheet, View } from 'react-native';
-import { WebView } from 'react-native-webview';
+import {ActivityIndicator, SafeAreaView, StyleSheet, View} from 'react-native';
+import {WebView} from 'react-native-webview';
 import Header from '../../components/Header';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on importing the `COLORS` from the theme file. This will ensure consistency in color usage across the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->